pages/icons.js

```diff
@@ -0,0 +1,76 @@
+import Head from "next/head";
+import { useState } from "react";
+import * as FaIcons from "react-icons/fa";
+import * as MdIcons from "react-icons/md";
+import IconCard from "../components/IconCard";
+
+export default function Icons() {
+  const [searchedIconNames, setSearchedIconNames] = useState([]);
+  const [notFound, setNotFound] = useState();
+
+  /**
+   creating an icons object to map the original name of the icon as value
+   the icons name are in camelCase so converting it into lowercase for searchig
+   after searching we need the camelcase string to get the icon from react-icons
+   eg: icons = { mdoutlinetoggleon: 'MdOutlineToggleOn'}
+ */
+
+  const icons = {};
+
+  Object.keys(FaIcons).forEach((key) => {
+    icons[key.toLocaleLowerCase()] = key;
+  });
+
+  Object.keys(MdIcons).forEach((key) => {
+    icons[key.toLocaleLowerCase()] = key;
+  });
+
+  const searchIcons = (value) => {
+    setSearchedIconNames([]);
+    if (value.length < 3) {
+      return setNotFound("Please enter at least 3 characters to search...");
+    }
+    const filteredIconNames = Object.keys(icons)
+      .filter((icon) => icon.includes(value.toLocaleLowerCase()))
+      .map((iconName) => icons[iconName]);
+
+    setSearchedIconNames(filteredIconNames);
+
+    if (!filteredIconNames.length) {
+      return setNotFound("No Icons found");
+    }
+
+    setNotFound();
+  };
+
+  return (
+    <>
+      <Head>
+        <title>LinkFree Search Icons</title>
+        <meta name="description" content="Search LinkFree icon directory" />
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+      <div className="flex flex-col px-20 pt-10 align-center">
+        {notFound && (
+          <h2 className="bg-red-200 text-red-600 border-2 border-red-600 p-5 my-5 text-xl">
```

Comment:
I think it would be better if we kept it consistent wit the `search` page https://github.com/EddieHubCommunity/LinkFree/blob/main/pages/search.js#L100

For example...
- the style of the alert box
- the position of it, below the input

Reply:
Yes updated the icons search page same as search page.

![image](https://user-images.githubusercontent.com/69904519/209457633-8c39ab48-7653-42f9-bc5b-84d9c709a08f.png)

![image](https://user-images.githubusercontent.com/69904519/209457642-34a3d77b-1e97-4019-85bf-651be5b521cf.png)